src/serverlist.cpp

```diff
@@ -123,6 +127,16 @@ CServerListManager::CServerListManager ( const quint16  iNPortNum,
     qDebug() << "Using" << qhaServerPublicIP.toString() << "as external IP.";
     SlaveCurLocalHostAddress = CHostAddress ( qhaServerPublicIP, iNPortNum );
 
+    if ( bEnableIPv6 )
+    {
+        // set the server internal address, including internal port number
+        QHostAddress qhaServerPublicIP6;
+
+        qhaServerPublicIP6 = NetworkUtil::GetLocalAddress6().InetAddr;
+        qDebug() << "Using" << qhaServerPublicIP6.toString() << "as external IPv6.";
```

Comment:
I don't agree with this one. When IP addresses are shown (e.g. by `ip addr` or `host`), they are not quoted, as they are values in their own right, not just strings. It would be like displaying the numeric port number as `'22124'` instead of `22124`.

Reply:
> It would be like displaying the numeric port number as '22124' instead of 22124.

Yes. It’s just cosmetics/personal preference. Since we have spaces before and after the IP and they all follow a standard probably the quotes are not needed.